Outline updates (#200)
greenelab/covid19-review: content/20.treatments.md

-->

### 4.2 Small Molecule Drugs for Targeting SARS-CoV-2

The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps.
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication.
Importantly, SARS-CoV-2 is an RNA virus.
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

> <!--#### 4.2.a Nucleotide Analogs

Author: ```suggestion
#### 4.2.a Nucleotide Analogs
```

> Removing this header for now, if we add additional nucleotide analogs, we can put it back

Reviewer: I think both of the drugs we have in this section are nucleotide analogs or precursors.

Author: Oh no, I got lost while trying to make it easier to follow! Hopefully fixed now.

Why one might use nucleotide analogs.
-->

> #### 4.2.a Avigan

Author: ```suggestion
##### 4.2.a.i Avigan
```

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection.
Avigan is a nucleotide analog.

> ##### 4.2.a.i Anticipated Mechanism

Author: ```suggestion
###### 4.2.a.i.1 Anticipated Mechanism
```

Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

#### Nucleotide Analogs
Removing this header for now, if we add additional nucleotide analogs, we can put it back
Why one might use nucleotide analogs.

> -->

Author: ```suggestion
```

Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007].
However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation.

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of the antiviral approach.

#### Remdesivir

> Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola.

Author: ```suggestion
Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola Virus Disease.
```

It does not have any FDA-approved use.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued [@doi:10.1056/NEJMoa1910993].

##### Anticipated Mechanism

Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18; @doi:10.1038/s41422-020-0282-0].

> Although it was developed against the Ebola Virus Disease, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].

Author: ```suggestion
Although it was developed against Ebola, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
```

Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007].
However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation.

> ##### 4.2.a.ii Current Evidence

Author: ```suggestion
###### 4.2.a.i.2 Current Evidence
```

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

> ##### 4.2.a.iii Summary

Author: ```suggestion
###### 4.2.a.i.3 Summary
```

Summarize the state of the antiviral approach.

> #### 4.2.b Remdesivir

Author: ```suggestion
##### 4.2.a.ii Remdesivir
```

Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola.
It does not have any FDA-approved use.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued [@doi:10.1056/NEJMoa1910993].

> ##### 4.2.b.i Anticipated Mechanism

Author: ```suggestion
###### 4.2.a.ii.1 Anticipated Mechanism
```

Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18; @doi:10.1038/s41422-020-0282-0].

> Although it was developed against Ebola, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].

Author: ```suggestion
Although it was developed against Ebola Virus Disease, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
```

It also inhibits SARS-CoV-2, showing synergy with chloroquine [@doi:10.1038/s41422-020-0282-0].

> ##### 4.2.b.ii Current Evidence

Author: ```suggestion
###### 4.2.a.ii.2 Current Evidence
```

All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
Remdesivir recently entered controlled clinical trials, and as of March 2020, there are six clinical trials underway to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs [@doi:10.1038/s41422-020-0282-0; @url:https://clinicaltrials.gov/ct2/show/NCT04292730; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @url:https://clinicaltrials.gov/ct2/show/NCT04252664; @url:https://clinicaltrials.gov/ct2/show/NCT04257656].

> ##### 4.2.b.iii Summary

Author: ```suggestion
###### 4.2.a.ii.3 Summary
```

In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   

> #### 4.2.c Protease Inhibitors

Author: ```suggestion
#### 4.2.b Protease Inhibitors
```

Why it may be useful

<!--#### Protease Inhibitor 1

> ##### 4.2.c.i Anticipated Mechanism

Author: ```suggestion
##### 4.2.b.i Anticipated Mechanism
```

> ##### 4.2.c.ii Current Evidence

Author: ```suggestion
##### 4.2.b.ii Current Evidence
```

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

> ##### 4.2.c.iii Summary

Author: ```suggestion
##### 4.2.b.iii Summary
```

Summarize the state of the antiviral approach.
-->

> #### 4.2.d Molecules Targeting the Viral Envelope

Author: ```suggestion
#### 4.2.c Molecules Targeting the Viral Envelope
```

Why it may be useful

<!--##### Viral Envelope Targeter 1

> ##### 4.2.d.i Anticipated Mechanism

Author: ```suggestion
##### 4.2.c.i Anticipated Mechanism
```

> ##### 4.2.d.ii Current Evidence

Author: ```suggestion
##### 4.2.c.ii Current Evidence
```

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

> ##### 4.2.d.iii Summary

Author: ```suggestion
##### 4.2.c.iii Summary
```

Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001)  [@doi:10.3390/nu11040708]. 
Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].

###### 4.3.b.iii.c Summary

The evidence suggests that vitamin C supplementation can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in ICU when administered at high doses.
While there is anecdotal evidence from China and the United States that vitamin C is currently being used for the treatment of COVID-19,  further controlled research is required.

> ##### 4.3.b.iv Vitamin D

Reviewer: Is there any way to generate these automatically (maybe tagging @agitter or @dhimmel here)? I think that maintaining these header identifiers by hand might turn out to be brutal. Having them done automatically can also help to show where a header level is not as intended.

Reviewer 2: Pandoc's manual states:
![image](https://user-images.githubusercontent.com/3532898/79226821-9ca1e700-7e24-11ea-90c3-cf0b0a86dc6d.png)

I haven't tested this.  My reading is that automatic numbering is supported, in which case I definitely agree we should use it instead of manually updating these.  I'm not sure why only HTML is supported.  Hopefully because we're using athenapdf to convert the HTML manuscript to PDF, we would get the section numbers.

@rando2 if you'd like me to explore numbering, you can create a new issue and assign it to me to work on post-merge.

Summarize the state of interferons.

### Vaccines 

#### Strategies for and challenges to vaccine development

> Generally, the first step in producing a vaccine is characterizing the target.

Reviewer 3: 'target' is an ambiguous term here. also, if you mean a gene target, this is not true historically. e.g. small pox.

Summarize the state of interferons.

### Vaccines 

#### Strategies for and challenges to vaccine development

> Generally, the first step in producing a vaccine is characterizing the target.

Author: ```suggestion
Today, the first step in producing a vaccine often is characterizing the target.
```
Per @ypar's suggestion!
